### PR TITLE
Add Badges in component library

### PR DESCRIPTION
**Pull request by nikhilmalviya-dev**
Add different types of badges in component library

### CODE REVIEW

File: Badges/badge.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+.amitabh-badge{
+    width: 100px;
+    height: 100px;
```
Comment (Jia-A) [inline Badges/badge.css]: change values from px to rem

File: Badges/badge.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+.code{
+    padding: 3rem 0rem;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    background-color:rgb(139, 185, 179);
+    color:black;
+    font-weight: bold;
+    font-size:19px;
```
Comment (Jia-A) [inline Badges/badge.css:24]: px to rem

File: Badges/badge.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+.side-heading{
+    font-size:1.8rem;
+    font-weight: bold;
+    margin-bottom:-0.5rem;
+}
+#special-btn{
+    color: black;
+    text-decoration: underline;
+}
+.componant{
```
Comment (Jia-A) [inline Badges/badge.css:66]: your "component" spelling is incorrect, not that it matters much but still.

File: Badges/badge.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+.avtar-badge{
+    position: relative;
+}
+.badge1{
+    text-align: center;
+    width: 1.4rem;
+    height: 1.4rem;
+    background-color:rgb(160, 14, 14);
+    position:absolute;
+    right:520px;
```
Comment (Jia-A) [inline Badges/badge.css]: change px to rem, it might make your site more responsive.

**Review by Jia-A — COMMENTED** [review]
Everything else looks good to me. Just change the values from px to rems.

**Review by Jia-A — COMMENTED** [review]
Amazingly done!

File: Badges/badge.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    height: 6rem;
+    border-radius:50%;
+}
+.colorW{
+    color: white;
+}
+.radius{
+    border-radius:50%;
+}
+button{
```
Comment (junaidshaikh-js) [inline Badges/badge.css:12]: can be moved to reset css  file if you have one

**Review by Bharatrathod-81 — APPROVED** [review]
Able to merge

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me

**Pull request merged**